dev-util/rt-tests/rt-tests-1.5.ebuild

```diff
@@ -0,0 +1,33 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+PYTHON_COMPAT=(python3_{5,6,7})
+
+inherit python-single-r1
+
+DESCRIPTION="A collection of latency testing tools for the linux(-rt) kernel"
+HOMEPAGE="https://git.kernel.org/pub/scm/utils/rt-tests/rt-tests.git/about/"
+SRC_URI="https://kernel.org/pub/linux/utils/rt-tests/${P}.tar.xz"
+
+LICENSE="GPL-2 GPL-2+ LGPL-2.1+"
+SLOT="0"
+KEYWORDS="~amd64 ~arm ~arm64 ~mips ~ppc ~ppc64 ~x86"
```

Comment:
You can't add these keywords. You can only add amd64 and x86, provided you have **tested** this ebuild on these arches. You'll have to request keywording through bugzilla for any other arch.

Reply:
I see. Thanks.